### PR TITLE
don't use cookies, use session storage

### PR DESCRIPTION
**Pull request by giannif**
remove the `giphy_pbid` cookie, and generate a 16 digit id on the client, saved in session storage

### CODE REVIEW

File: packages/fetch-api/src/api.ts [hunk 1]
```diff
@@ -34,9 +34,7 @@ export class GiphyFetch {
      * @hidden
      */
     private getQS = (options: any = {}) => {
-        const { giphy_pbid: pingback_id } =
-            typeof document !== 'undefined' ? cookie.parse(document.cookie) : ({} as any)
-        return qs.stringify({ ...options, api_key: this.apiKey, pingback_id })
+        return qs.stringify({ ...options, api_key: this.apiKey, pingback_id: getPingbackId() })
```
Comment (giannif) [inline packages/fetch-api/src/api.ts:37]: Remove the cookie, use the getPingbackId util

File: packages/util/src/get-pingback-id.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const getPingbackId = () => {
+    // it exists in memory
+    if (!pingbackId) {
+        try {
+            // it exists in storage
+            pingbackId = sessionStorage.getItem('giphyPingbackId') as string
+        } catch (_) {}
+        if (!pingbackId) {
+            // we need to create it
+            pingbackId = uuid().replace(/-/g, '').substring(0, 16)
```
Comment (giannif) [inline packages/util/src/get-pingback-id.ts:12]: keep the uuid the required length 

@seanpquig 16 length doesn't ensure uniqueness 100%, but assuming that's okay

File: packages/fetch-api/src/api.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /* eslint-disable no-dupe-class-members */
-import cookie from 'cookie'
+import { getPingbackId } from '@giphy/js-util'
```
Comment (giannif) [inline packages/fetch-api/src/api.ts:2]: @cosmocochrane1 you can get the pingback id in web app like so

File: packages/fetch-api/src/api.ts (hunk 1 above)
Comment (cosmocochrane1) [reply packages/fetch-api/src/api.ts:37]: 🔥

File: packages/util/src/get-pingback-id.ts (hunk 2 above)
Comment (cosmocochrane1) [reply packages/util/src/get-pingback-id.ts:12]: ooooo

File: packages/fetch-api/src/api.ts (hunk 3 above)
Comment (cosmocochrane1) [reply packages/fetch-api/src/api.ts:2]: 👍

**Review by cosmocochrane1 — APPROVED** [review]

File: packages/util/src/get-pingback-id.ts (hunk 2 above)
Comment (cosmocochrane1) [reply packages/util/src/get-pingback-id.ts:12]: might need to add time badck into the mix

Comment (giannif) [reply packages/util/src/get-pingback-id.ts:12]: @cosmocochrane1 @seanpquig if it needs to be 16 digits, we can't use uuid, but we can just generate a random number using a lib, something from here: https://www.npmjs.com/search?q=random+number 

lmk if you want to use a random number that is 16 digits and I'll make it happen with the first option in npm called `brorand`

**Pull request merged**